sycl/cmake/modules/FetchUnifiedRuntime.cmake

```diff
@@ -116,7 +116,7 @@ if(SYCL_UR_USE_FETCH_CONTENT)
       CACHE PATH "Path to external '${name}' adapter source dir" FORCE)
   endfunction()
 
-  set(UNIFIED_RUNTIME_REPO "https://github.com/oneapi-src/unified-runtime.git")
+  set(UNIFIED_RUNTIME_REPO "https://github.com/kurapov-peter/unified-runtime.git")
```

Comment:
This is needed until https://github.com/oneapi-src/unified-runtime/pull/2614 is merged.